### PR TITLE
Separate libunwind into 1.3.2 and 1.5.0 builders, add a patch

### PR DESCRIPTION
**Pull request by ararslan**
As of #3804, the libunwind builder builds v1.5.0. However, Julia still uses v1.3.2 (cf. https://github.com/JuliaLang/julia/pull/42782). In order to fix https://github.com/JuliaLang/julia/issues/41943 we'll need https://github.com/libunwind/libunwind/pull/308, but that fix will also need to be backported. Thus it seems the path of least resistance is to add back the previous v1.3.2 builder (the contents of the L/LibUnwind directory prior to #3804) and create new JLL releases for both libunwind versions.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -59,6 +59,8 @@ jobs:
                   dt = tryparse(DateTime, replace(last_mod, " GMT"=>""), dateformat"e, d u y H:M:S")
                   # If parsing did fail, set the delay to 0.
                   delay = isnothing(dt) ? Second(0) : now(UTC) - dt
+              else
+                  delay = Second(0)
```
Comment (giordano) [inline azure-pipelines.yml:63]: I cherry-picked this in #3858, since this is blocking everything at the moment.

**Review by giordano — CHANGES_REQUESTED** [review]
Looks good overall, but I'd suggest to reorganise this a bit, to avoid duplication:

* have a file `L/LibUnwind/common.jl` with most variables defined there, in particular the `script`.  We have plenty of similar setups for other Julia dependencies, like LLVM and OpenBLAS
* create the 1.5.0 directory in a separate pull request, otherwise registration is going to fail because of merge conflicts, which are a mess to manually fix when involve different compat bounds.

I know this is an extra hassle now, but we'll make it way easier to add more versions later on.

**Review by giordano — APPROVED** [review]

**Comment by DilumAluthge** [comment]
Thank you @ararslan and @giordano for getting this done so quickly!

**Comment by ararslan** [comment]
Note: This PR was restructured to just be 1.5.0, and I'll be submitting a follow-up with 1.3.2.

**Pull request merged**